src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -173,6 +173,11 @@ private VariableInformation(ThreadContext ctx, AD7Engine engine, AD7Thread threa
             _attribsFetched = false;
             Access = enum_DBG_ATTRIB_FLAGS.DBG_ATTRIB_NONE;
             _fullname = null;
+
+            lock(this._debuggedProcess.ActiveVariables)
+            {
+                this._debuggedProcess.ActiveVariables.Add(this);
```

Comment:
Does it make sense to add all the children since it doesn't do much of anything?